app/controllers/alchemy/admin/resources_controller.rb

```diff
@@ -102,7 +100,11 @@ def flash_notice_for_resource_action(action = params[:action])
         when :destroy
           verb = "removed"
         end
-        flash[:notice] = Alchemy.t("#{resource_handler.resource_name.classify} successfully #{verb}", default: Alchemy.t("Successfully #{verb}"))
+        if resource_instance_variable.errors.any?
```

Comment:
This will render the error messages twice for any regular form. What about only making this on a non-renderable action (aka. destroy)

Reply:
Done!